internal/apps/dop/endpoints/iteration.go

```diff
@@ -205,6 +206,7 @@ func (e *Endpoints) DeleteIteration(ctx context.Context, r *http.Request, vars m
 	}
 
 	iterationID, err := strconv.ParseUint(vars["id"], 10, 64)
+	onlyIteration := r.URL.Query().Get("onlyIteration")
```

Comment:
parse `onlyIteration` string to bool here

Reply:
ok,thanks